### ISSUE TITLE
[data request] ImageNet21k

### ISSUE THREAD

**Issue by lgeiger** [tensorflow/datasets#3414]
* Name of dataset: ImageNet21k
* URL of dataset: https://github.com/Alibaba-MIIL/ImageNet21K, https://image-net.org/download-images.php
* License of dataset: [Same as ImageNet 2012](https://image-net.org/download.php)
* Short description of dataset and use case(s):
   ImageNet-21K is an extension to the popular ImageNet ILSVRC 2012 dataset. The train set contains 11060223 samples, test set contains 522500 samples with 10450 classes. ImageNet (including many variations of it) is already part of TFDS so I think it would make sense to include ImageNet-21K as well given that it has been used for pretraining in many recent computer vision papers.

Folks who would also like to see this dataset in `tensorflow/datasets`, please thumbs-up so the developers can know which requests to prioritize.

And if you'd like to contribute the dataset (thank you!), see our [guide to adding a dataset](https://github.com/tensorflow/datasets/blob/master/docs/add_dataset.md).

**Comment by jrabary** [comment]
Any progress on this issue ? We have downloaded the raw dataset but it is really huge. We need computational power to generate de TFDS version. Having the tfrecords ready for download would be easy way for most of us.
